spark-operator/src/main/java/org/apache/spark/k8s/operator/config/SparkOperatorConf.java

```diff
@@ -226,9 +226,9 @@ public class SparkOperatorConf {
           .defaultValue(-1)
           .build();
 
-  public static final ConfigOption<Integer> RETRY_MAX_ATTEMPTS =
+  public static final ConfigOption<Integer> API_RETRY_MAX_ATTEMPTS =
       ConfigOption.<Integer>builder()
-          .key("spark.kubernetes.operator.retry.maxAttempts")
+          .key("spark.kubernetes.operator.api.retryMaxAttempts")
```

Comment:
New namespace `spark.kubernetes.operator.api` could be misleading.

This is the backend K8s API-related configurations, right? If then, can we rename this more specifically in order to leave a room for `Apache Spark Kubernetes Operator`'s API-related configurations?